packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -86,6 +94,7 @@ export class ApexDebug extends DebugSession {
   ): void {
     this.myBreakpointService.clearSavedBreakpoints();
     this.initializedResponse = response;
+    this.sendEvent(new Event(GET_PROXY_SETTINGS_EVENT));
```

Comment:
Requesting proxy settings from the vscode-apex-debugger module since it's accessible through vscode.workspace.